sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -1,7 +1,54 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2020-02-11)
 
+**Breaking changes**
+
+- The single text, module-level operations return an atomic type of the operation result. For example, `detectLanguage(String text)` returns a `DetectedLanguage` rather than a `DetectLanguageResult`.
+
+  For other module-level operations:
+    
+  `recognizeEntities(String text)`, it no longer returns type of `Mono<RecognizeEntitiesResult>` but `PagedFlux<CategorizedEntity>`.
+  
+  `recognizePiiEntities(String text)`, it no longer returns type of `Mono<RecognizePiiEntitiesResult>` but `PagedFlux<PiiEntity>`.
+  
+  `recognizeLinkedEntities(String text)`, it no longer returns type of `Mono<RecognizeLinkedEntitiesResult>` but `PagedFlux<LinkedEntity>`.
+  
+  `extractKeyPhrases(String text)`, it no longer returns type of `Mono<ExtractKeyPhraseResult>` but `PagedFlux<String>`.
+  
+  `analyzeSentiment(String text)`, it no longer returns type of `Mono<AnalyzeSentimentResult>` but `Mono<DocumentSentiment>`.
+  
+  `recognizeEntitiesWithResponse(String text, String language)` changed to `recognizeEntities(String text, String language)` and return `PagedFlux<CategorizedEntity>` as a collection of atomic type `CategorizedEntity`.
+  
+  `recognizePiiEntitiesWithResponse(String text, String language)` changed to `recognizePiiEntities(String text, String language)` and return `PagedFlux<PiiEntity>` as a collection of atomic type `PiiEntity`.
+  
+  `recognizeLinkedEntitiesWithResponse(String text, String language)` changed to `recognizeLinkedEntities(String text, String language)` and return `PagedFlux<LinkedEntity>` as a collection of atomic type `LinkedEntity`.
+  
+  `extractKeyPhrasesWithResponse(String text, String language)` changed to `extractKeyPhrases(String text, String language)` and return `PagedFlux<String>` as a collection of atomic type `String`.
+  
+  `analyzeSentimentWithResponse(String text, String language)` return `Mono<Response<DocumentSentiment>>` with an atomic type `DocumentSentiment`.
+
+- Removed `TextSentiment` class but created `DocumentSentiment` and `SentenceSentiment` instead. `DocumentSentiment` includes a list of `SentenceSentiment`.
+
+- Added a new class model `SentimentScorePerLabel` for the scores of sentiment label.  
+- Added a new parameter `TextAnalyticsRequestOptions options` to method overloads accepting a list of text inputs for allowing the users to opt for batch operation statistics.
+- Passing the API key as a string is no longer supported. To use subscription key authentication a new credential class `TextAnalyticsApiKeyCredential("<api_key>")` must be passed in for the `credential` parameter.
+- `detectLanguages()` is renamed to `detectLanguage()`.
+- The `TextAnalyticsError` model has been simplified to an object with only attributes `code`, `message`, and `target`.
+- `RecognizePiiEntitiesResult` now contains on the object a list of `PiiEntity` instead of `NamedEntity`.
```

Comment:
The same is true for `RecognizeLinkedEntitiesResult`

Reply:
oh I saw it.... I guess it deserves its own bullet point. otherwise it is confusing